dag/set_test.go

```diff
@@ -119,3 +119,13 @@ func TestSetCopy(t *testing.T) {
 	}
 
 }
+
+func BenchmarkSetIntersectionLargeToSmall(b *testing.B) {
+	var small, large Set
```

Comment:
This panics for me—the `Set` needs to be initialized with `make(Set)` to allow it to be used.

Reply:
yes, sorry I work on a franken-version from a couple releases ago... I just used your code, which is clearly better.